plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryTableHandle.java

```diff
@@ -118,10 +113,10 @@ public boolean equals(Object o)
             return false;
         }
         BigQueryTableHandle that = (BigQueryTableHandle) o;
-        return Objects.equals(projectId, that.projectId) &&
-                Objects.equals(schemaName, that.schemaName) &&
-                Objects.equals(tableName, that.tableName) &&
-                Objects.equals(type, that.tableName) &&
+        // NOTE remoteTableName is not compared here because two handles differing in only remoteTableName will create ambiguity
```

Comment:
you may want to address https://github.com/trinodb/trino/blob/75baaaed7c03bb504aa69cf664e92f15193b23d0/plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcNamedRelationHandle.java#L63 in follow up

Reply:
Oh, what a nice coincidece. It looks like BaseJdbc has some equivalence tests too. I'll address in a follow-up PR.

I'm creating an issue with a checklist of things I've found that need some attention (some of which are good beginner issues).